### PR TITLE
Feat: add docker-compose with prod requisites

### PR DESCRIPTION
**Pull request by Aletor93**
Here, I'll add : 
- Traefik config
- Logging config
- Volume rework (still needs to be improved)

The actual `docker-compose.yaml` should be renamed `docker-compose.local.yaml` (or `docker-compose.dev.yaml` or whatever..).
And then this docker-compose file should be renamed `docker-compose.yaml` instead of `docker-compose.prod.yaml`.

### CODE REVIEW

File: docker-compose.prod.yaml [hunk 1]
```diff
@@ -2,9 +2,24 @@ version: "3.2"
 services:
   forseti:
     image: navitia/forseti:master
```
Comment (azime) [inline docker-compose.prod.yaml:4]:
```suggestion
    image: navitia/forseti:latest OR  image: navitia/forseti:release
```

**Review by azime — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

File: docker-compose.yaml [hunk 2]
```diff
@@ -37,11 +52,3 @@ services:
       - FORSETI_OCCUPANCY_SERVICE_TOKEN=
       - FORSETI_OCCUPANCY_REFRESH=300s
       - FORSETI_ROUTESCHEDULE_REFRESH=300s
```
Comment (kadhikari) [inline docker-compose.yaml:54]:
```suggestion
      - FORSETI_ROUTESCHEDULE_REFRESH=86400s
```

File: docker-compose.local.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      # Occupancies parameters
+      - FORSETI_OCCUPANCY_SERVICE_REFRESH_ACTIVE=False
+      - FORSETI_OCCUPANCY_FILES_URI=file:///data
+      - FORSETI_OCCUPANCY_NAVITIA_URI=
+      - FORSETI_OCCUPANCY_SERVICE_URI=
+      - FORSETI_OCCUPANCY_NAVITIA_TOKEN=
+      - FORSETI_OCCUPANCY_SERVICE_TOKEN=
+      - FORSETI_OCCUPANCY_REFRESH=300s
+      - FORSETI_ROUTESCHEDULE_REFRESH=300s
```
Comment (kadhikari) [inline docker-compose.local.yaml:39]:
```suggestion
      - FORSETI_ROUTESCHEDULE_REFRESH=86400s
```

**Review by kadhikari — CHANGES_REQUESTED** [review]
We don't want to reload routeschedule every 5 minutes (300s), every hour will be enough

**Review by kadhikari — APPROVED** [review]

**Comment by Aletor93** [comment]
I'm not able to reach `/status` page provided with the image through the loadbalancer (Traefik).
I made some researches but didn't found anything yet. 

Forseti logs are available [here](http://ctp-int-logs.canaltp.prod/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(_source),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'673058a0-7806-11ea-b6df-9b3c3277ec8d',key:'@log_name',negate:!f,params:(query:forseti.departure),type:phrase),query:(match_phrase:('@log_name':forseti.departure)))),index:'673058a0-7806-11ea-b6df-9b3c3277ec8d',interval:auto,query:(language:kuery,query:forseti),sort:!())) for the Internal deployment.

**Review by benoit-bst — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Comment by Aletor93** [comment]
Hello, could we merge this branch please ? :) I'll update to master branch in my code.

**Pull request merged**